Casks/propresenter-beta.rb

```diff
@@ -1,23 +1,40 @@
 cask "propresenter-beta" do
-  version "7.15,118423567"
-  sha256 "70e71ec898d16bf54852470255ae31f4edb095fa5441cf5fb2602486853e35de"
+  on_big_sur :or_older do
+    version "7.15,118423570"
```

Comment:
```suggestion
    version "7.15,118423570"
```

Is this supposed to be `7.14.1,118358275` (i.e., the newest version from this `livecheck` block)?

Reply:
@samford Yes that's correct. And definitely good to move to parsing the JSON, we can migrate the logic to the main cask too.